include/tachimawari/joint.hpp

```diff
@@ -35,28 +35,29 @@ class Joint
 
   void set_target_position(float target_position, float speed = 1.0);
   void set_target_position(float present_position, float target_position, float speed);
+  void set_simulator_target_position(float target_position, float speed);
   void set_present_position(float present_position);
   void set_pid_gain(float p, float i, float d);
```

Comment:
If possible, it's better to reinforce pass by constant reference (`const T & foo`) in the function parameters which is not a pointer.